activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -285,6 +301,12 @@ def utc_offset
       end
     end
 
+    # Returns the offset of this time zone from UTC in seconds,
+    # taking DST into account at the given time.
+    def utc_total_offset(time=Time.now, dst=true)
```

Comment:
Add a space before and after the argument equals sign: `def utc_total_offset(time = Time.now, dst = true)`.

Let's also convert `dst` to a keyword argument, so it's easier to read when calling the method: `utf_total_offset(today, dst: true)`